health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/BoundaryRelationCreate.js

```
@@ -340,14 +340,22 @@ const BoundaryRelationCreate = () => {
-            const bh = [...boundaryData, ...newBoundaryData];
-            const local = bh.map(item => ({
-                code: `${hierarchyType}_${item.boundaryType}`.toUpperCase(),
-                message: item.boundaryType,
-                module: `hcm-boundary-${hierarchyType.toLowerCase()}`,
-                locale: locale
-            }));
+
+            const local = [
+                ...boundaryData.map(item => ({
+                    code: `${hierarchyType}_${item.boundaryType.trim().replace(/[\s_]+/g, '')}`.toUpperCase(),
+                    message: `${t((defaultHierarchyType + "_" + item?.boundaryType).toUpperCase())}`,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)